api/src/main/scala/uk/ac/wellcome/platform/api/models/DisplayLicense.scala

```diff
@@ -0,0 +1,26 @@
+package uk.ac.wellcome.platform.api.models
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import io.swagger.annotations.{ApiModel, ApiModelProperty}
+
+@ApiModel(
+  value = "License",
+  description =
+    "The specific license under which the work in question is released to the public - for example, one of the forms of Creative Commons - if it is a precise license to which a link can be made."
+)
+case class DisplayLicense(
+  @ApiModelProperty(
+    value =
+      "A type of license under which the work in question is released to the public.",
+    allowableValues = "CC-BY, CC-BY-NC"
```

Comment:
I really want these to be dynamically populated from JSON files that live externally – that’s coming in a separate patch. Happy to leave as hard-coded for now?